less/home-page.less

```diff
@@ -56,20 +68,106 @@
       p {
         margin-bottom: 26px;
       }
-
-      .btn {
-      }
+      
+      
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
+      max-width: 100%;
       
       img {
-        width: 100%;
+       width: 100%;
         height: auto;
         border-radius: 10px;
       }
     }
+    
+  }
+   // added mixins to my button
+  .btn{
+    .button();
+    animation-name: spin;
+    animation-duration: 8s;
+
+  }
+  // added animation
+  @keyframes spin{
+    from{transform: rotate(0deg);}
+    to{transform: rotate(360deg);}
+}
+  
+}// home
+
+
+// media queries start here
+@media @mobile{
+  
+  .walk-img {
+     
+   width: 100%;
+    
+    
+   #walk-img{
+     width: 100%;
+   }
+  }
+    
+  .space-images #launch-img{
+    max-width: 500px;
+    /*max-height: 500px; */
+    width: 100%;
+    /*height: 100%; */
+    margin: 10px;
+    padding: 0;
+    
+    
+
+  }
+  
+  .space-images :nth-last-child(1){
+    display: none;
+  }
+  .space-images :nth-child(2){
```

Comment:
Good use of psuedo selector tograb the images here, but you don't actually need the `1` in the parentheses on line 129.